tests/unittests/test_net.py

```diff
@@ -2702,236 +2703,6 @@ def sys_dev_path(devname, path=""):
-            return_value="root=/dev/sda1")
-
-    @mock.patch("cloudinit.net.sys_dev_path")
-    @mock.patch("cloudinit.net.read_sys_net")
-    @mock.patch("cloudinit.net.get_devicelist")
-    def test_device_driver_v2(self, mock_get_devicelist, mock_read_sys_net,
-                              mock_sys_dev_path):
-        """Network configuration for generate_fallback_config is version 2."""
-        devices = {
-            'eth0': {
-                'bridge': False, 'carrier': False, 'dormant': False,
-                'operstate': 'down', 'address': '00:11:22:33:44:55',
-                'device/driver': 'hv_netsvc', 'device/device': '0x3',
-                'name_assign_type': '4'},
-            'eth1': {
-                'bridge': False, 'carrier': False, 'dormant': False,
-                'operstate': 'down', 'address': '00:11:22:33:44:55',
-                'device/driver': 'mlx4_core', 'device/device': '0x7',
-                'name_assign_type': '4'},
-
-        }
-
-        tmp_dir = self.tmp_dir()
-        _setup_test(tmp_dir, mock_get_devicelist,
-                    mock_read_sys_net, mock_sys_dev_path,
-                    dev_attrs=devices)
-
-        network_cfg = net.generate_fallback_config(config_driver=True)
-        expected = {
-            'ethernets': {'eth0': {'dhcp4': True, 'set-name': 'eth0',
-                                   'match': {'macaddress': '00:11:22:33:44:55',
-                                             'driver': 'hv_netsvc'}}},
-            'version': 2}
-        self.assertEqual(expected, network_cfg)
-
-    @mock.patch("cloudinit.net.sys_dev_path")
-    @mock.patch("cloudinit.net.read_sys_net")
-    @mock.patch("cloudinit.net.get_devicelist")
-    def test_device_driver(self, mock_get_devicelist, mock_read_sys_net,
-                           mock_sys_dev_path):
-        devices = {
-            'eth0': {
-                'bridge': False, 'carrier': False, 'dormant': False,
-                'operstate': 'down', 'address': '00:11:22:33:44:55',
-                'device/driver': 'hv_netsvc', 'device/device': '0x3',
-                'name_assign_type': '4'},
-            'eth1': {
-                'bridge': False, 'carrier': False, 'dormant': False,
-                'operstate': 'down', 'address': '00:11:22:33:44:55',
-                'device/driver': 'mlx4_core', 'device/device': '0x7',
-                'name_assign_type': '4'},
-
-        }
-
-        tmp_dir = self.tmp_dir()
-        _setup_test(tmp_dir, mock_get_devicelist,
-                    mock_read_sys_net, mock_sys_dev_path,
-                    dev_attrs=devices)
-
-        network_cfg = net.generate_fallback_config(config_driver=True)
-        ns = network_state.parse_net_config_data(network_cfg,
-                                                 skip_broken=False)
-
-        render_dir = os.path.join(tmp_dir, "render")
-        os.makedirs(render_dir)
-
-        # don't set rulepath so eni writes them
-        renderer = eni.Renderer(
```

Comment:
@raharper any reason why we're mostly testing the renderers here in these tests?

Reply:
This unittest file has always been about rendering network config, even before we had rendering classes.

See 44f71ff4cc and look at tests/unittests/test_net.py

Reply:
so, should i keep these tests and try to rework them for the refactor?